.github/workflows/sdk-update-test.yml

```diff
@@ -91,14 +91,25 @@ jobs:
           include: "code_examples/**"
           exclude: "**/*.lock"
           regex: true
-      - name: Regenerate yarn lock
+      - uses: jacobtomlinson/gha-find-replace@v2
```

Comment:
pls don't remove the vitejs locking when fixing merge conflicts

Reply:
I will go ahead and merge this, but can you maybe double check when you're back that I've done it correctly?

Reply:
looks good!